app/src/main/java/com/example/checkpoint/LocationPermissionsHelper.kt

```diff
@@ -24,6 +24,10 @@ class LocationPermissionHelper(val activity: WeakReference<Activity>) {
                 override fun onPermissionResult(granted: Boolean) {
                     if (granted) {
                         onMapReady()
+                        Toast.makeText(
+                            activity.get(), "Using your current location. ",
+                            Toast.LENGTH_SHORT
```

Comment:
Let the user know that their location is being used by the app for Accessibility

Reply:
Avoid over-toasting.  That annoys the user.  
The user will know it's his/her current location by looking at the map.